### ISSUE TITLE
Tests are broken with Undefined variable: legacyCampaignId

### ISSUE THREAD

**Issue by sergiitk** [DoSomething/gateway#79]
```
❯ ./vendor/phpunit/phpunit/phpunit
PHPUnit 4.8.35 by Sebastian Bergmann and contributors.

.....E................

Time: 204 ms, Memory: 11.00MB

There was 1 error:

1) GladiatorTest::testStoringAUser
Undefined variable: legacyCampaignId

/Users/sergii/Development/gateway/src/Gladiator.php:56
/Users/sergii/Development/gateway/tests/GladiatorTest.php:25

FAILURES!
Tests: 22, Assertions: 36, Errors: 1.
```

**Comment by sergiitk** [comment]
Updated PHPUnit 4.8.36 also fails with the same error.

**Comment by DFurnes** [comment]
Oh no! I wonder how that one snuck in there, CI still shows green.

**Comment by sergiitk** [comment]
Yes, this is strange. I think it might be related to PHP version, testing this theory now.

**Comment by sergiitk** [comment]
PHP in wrecker is pinned to `php:latest`, which currently resolves to 7.1.8. I have OSX preinstalled php 5.6, will update it now.

**Comment by sergiitk** [comment]
Nope, not that.

```
❯ php -v
PHP 7.1.8 (cli) (built: Aug  5 2017 11:27:09) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2017 Zend Technologies
❯ ./vendor/phpunit/phpunit/phpunit
PHPUnit 4.8.36 by Sebastian Bergmann and contributors.

.....E................

Time: 75 ms, Memory: 8.00MB

There was 1 error:

1) GladiatorTest::testStoringAUser
Undefined variable: legacyCampaignId

/Users/sergii/Development/gateway/src/Gladiator.php:56
/Users/sergii/Development/gateway/tests/GladiatorTest.php:25

FAILURES!
Tests: 22, Assertions: 36, Errors: 1.
```
